subprojects/testing-base/src/integTest/groovy/org/gradle/testing/TestExecutionBuildOperationsIntegrationTest.groovy

```diff
@@ -50,24 +49,6 @@ class TestExecutionBuildOperationsIntegrationTest extends AbstractIntegrationSpe
         assertJunit(rootTestOp, operations)
     }
 
-    @ToBeFixedForInstantExecution
-    def "emitsBuildOperationsForTestNgTests"() {
```

Comment:
why remove the test and not the annotation?

Reply:
It was a mistake. Recovered.